releases/v1.1.1-rc.1.toml

```diff
@@ -0,0 +1,33 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.1.0"
+
+pre_release = true
+
+preface = """\
+This is the first patch release for the `containerd` 1.1 release. This
+includes bug fixes related to CRI, image pull, and native snapshotter.
+
+## CRI Plugin
+Fixes for working set memory calculation and privileged container creation.
```

Comment:
Fixes for working set memory calculation, privileged container creation and image volume directory ownership